line-sdk/src/main/java/com/linecorp/linesdk/openchat/KotlinExtensions.kt

```diff
@@ -0,0 +1,16 @@
+package com.linecorp.linesdk.openchat
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.EditText
+
+fun EditText.setAfterTextChangedAction(textChangedAction: (String) -> Unit) {
```

Comment:
The detail implementation is to add a new listener every time this extension is called. I think it'd better rename `set` to `add`, the `set` will mislead users the second action will overwrite the first action.